stream_alert/alert_processor/outputs/output_base.py

```diff
@@ -252,6 +295,32 @@ def _get_request(cls, url, params=None, headers=None, verify=True):
         return requests.get(url, headers=headers, params=params,
                             verify=verify, timeout=cls._DEFAULT_REQUEST_TIMEOUT)
 
+    @classmethod
+    def _get_request_retry(cls, url, params=None, headers=None, verify=True):
+        """Method to return the json loaded response for this GET request
+        This method implements support for backoff to retry failed requests
+
+        Args:
+            url (str): Endpoint for this request
+            params (dict): Payload to send with this request
+            headers (dict): Dictionary containing request-specific header parameters
+            verify (bool): Whether or not the server's SSL certificate should be verified
+        Returns:
+            dict: Contains the http response object
+        Raises:
+            OutputRequestFailure
+        """
+        @retry_on_exception(cls._catch_exceptions())
+        def do_get_request():
+            """Decorator to perform the request with retry/backoff"""
```

Comment:
Can you update this docstring? This technically isn't a `decorator` and is a decorated nested function. Same with the other similar function below.